File: Esay/Majority Element.java
//https://leetcode.com/problems/majority-element/description/
class Solution {
    public int majorityElement(int[] nums) {
        int count = 0;
        int ele=nums[0];
        int n = nums.length;
        for(int i=0 ;i<n;i++){
            if(count == 0){
                count = 1;
                ele = nums[i];
            }
            else if(nums[i] == ele){
                count++;
            }
            else{
                count--;
            }
        }
        return ele;
    }
}

File: Easy/Majority Element.cpp
//https://www.naukri.com/code360/problems/majority-element_6783241
int majorityElement(vector<int> v) {
	// Write your code here
	int count=0;
	int ele=v[0];
	for(int i=0;i<v.size();i++){
		if(count==0){
			count=1;
			ele=v[i];
		}else if(v[i]==ele){
			count++;
		}else {
			count--;
		}
	}
	return ele;
}
